src/components/TaskList.tsx

```diff
@@ -22,12 +22,28 @@ export default TaskList;
 
 function ExpandableTaskItem({ item }: { item: TaskItem }) {
   const [isExpanded, setIsExpanded] = useState(false);
+  const [isChecked, setIsChecked] = useState(false);
```

Comment:
We can start loading the state with the value
```suggestion
  const prevState = JSON.parse(localStorage.getItem(`TASK_CHECKED:${item.id}`)) || false
  const [isChecked, setIsChecked] = useState(prevState);
```